tests/integration/test_spans.py

```diff
@@ -1862,10 +1862,10 @@ def summarize_outcomes(outcomes):
     if sample_rate == 1.0:
         spans = list(spans_consumer.get_spans(timeout=10, max_attempts=4))
         assert len(spans) == 4
-        outcomes = outcomes_consumer.get_outcomes(timeout=0.1)
+        outcomes = outcomes_consumer.get_outcomes(timeout=3)
         assert summarize_outcomes(outcomes) == {(16, 0): 4}  # SpanIndexed, Accepted
     else:
-        outcomes = outcomes_consumer.get_outcomes(timeout=10)
+        outcomes = outcomes_consumer.get_outcomes(timeout=3)
```

Comment:
The reason the `else` timeout was longer is that in the `if` branch, we're already waiting for 4 spans, so more time has passed. This is also why the test fails now. Maybe `(1, 10)` would already work.

Reply:
This is so annoying, they are not flaky at all locally. I didn't wanna set a too high timeout just because that actually increases the runtime of the tests.